osrf/vorc
# Task 4

How to test it:

1. Launch VORC with the world ready for the gymkhana task:

```
 roslaunch vorc_gazebo task_gymkhana.launch
```

2. Verify the output of the acoustic pinger:

```
rostopic echo /cora/sensors/pingers/pinger/range_bearing
```
You should see something like this:

```
---
header: 
  seq: 7
  stamp: 
    secs: 8
    nsecs: 259000000
  frame_id: "cora/pinger"
range: 141.901062012
bearing: 0.330635935068
elevation: -0.0557640381157
---
```

3. Now move the CoRa model next to the `116, -278` point (it's in the obstacle course). Your acoustic sensor should give you a much closer range:

```
---
header: 
  seq: 420
  stamp: 
    secs: 421
    nsecs: 672000000
  frame_id: "cora/pinger"
range: 14.0360584259
bearing: -0.902148127556
elevation: -0.335353553295
---
```
